### ISSUE TITLE
Sound artifacts on .NET Core 6 and x64 builds

### ISSUE THREAD

**Issue by TacoGuyAT** [yvanvds/yse-soundengine#26]
Hi, I've trying to use YSE in my .NET Core project but there were no releases since 2018 (1.0.66). On .NET Framework sound works great, but on .NET Core it produces some strange artifacts. Pre-Built libraries are only x86 and though I've compiled x64 version of Yse.dll, native examples started to behave like my attempts at running YSE with .NET Core project. Maybe I've done something wrong while compiling it, but there's no instructions on how to do that, so can't really say much, especially since YSE C# demo project running on .NET Core 6 instead of .NET Framework using the nuget package (1.0.66) produces the same effect.

https://user-images.githubusercontent.com/35270005/184539667-3c0eb8e7-962f-4a9e-ac28-d5cc56ed4fbd.mp4

(Attached only x64 native video, since the result in .NET Core is exactly the same)

P. S.: Not really useful, but eh: [YSElog.txt](https://github.com/yvanvds/yse-soundengine/files/9333458/YSElog.txt)

**Comment by yvanvds** [comment]
Yeah, there is a problem there. I noticed that too, last year. Thing is, although I keep telling myself I am going to fix this, a year has passed and I did not find the time to work on it. So I certainly cannot promise you now that I'll fix it soon.

It does not have anything to do with .NET core or framework. The actual audio processing happens in the dll, which is c++ code. It seems that the actual audio callbacks are taking way more time to calculate than they used to. This results in missed callbacks and jitter.

I thought it might be related to the Visual Studio version and compiler version used. I got as far as installing VS2017 again but that did not solve the problem. But maybe an even earlier version of VS could work, like VS2015 or 2012. 

Another thing to look at are windows updates. Maybe there's something there. A few years back I started on another app which also uses  PortAudio and it started having the same problem. But portaudio itself is pretty stable and other users aren't reporting a problem.

If I had more time I would contact the portaudio developers, but I am reluctant to do so because I know I don't have time to contribute with testing and such.

Suggestions are welcome.

**Comment by yvanvds** [comment]
I started working on this project again, but decided against writing wrappers in this project. I am working on one for dart (https://github.com/yvanvds/dart-yse) in a separate project. But I doubt I will be working on a c# one. I do expose the C Api in here though, so making wrappers for any language should be rather easy. Certainly with the help of agentic AI.

**Comment by TacoGuyAT** [comment]
In case anyone is looking for C# alternative - I made a small wrapper for Kira audio engine, not something complex, but fits my needs well and can be extended in future: https://github.com/TacoGuyAT/KiraSharp
